src/org/exist/collections/CollectionCache.java

```diff
@@ -38,13 +39,13 @@
  * 
  * @author wolf
  */
-public class CollectionCache extends LRUCache {
+public class CollectionCache extends LRUCache implements BrokerPoolService {
 
     private Object2LongHashMap names;
     private BrokerPool pool;
 
     public CollectionCache(BrokerPool pool, int blockBuffers, double growthThreshold) {
-        super(blockBuffers, 2.0, 0.000001, CacheManager.DATA_CACHE);
+        super(blockBuffers, 2.0, growthThreshold, CacheManager.DATA_CACHE);
```

Comment:
Not exactly previously the `growthThreshold` parameter was orphaned, and it was clear that that should not have been the case.